src/Sylius/Bundle/ResourceBundle/spec/Form/Type/ResourceTranslationsTypeSpec.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\ResourceBundle\Form\Type;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Bundle\ResourceBundle\Form\EventSubscriber\ResourceTranslationsSubscriber;
+use Sylius\Component\Locale\Context\LocaleContextInterface;
+use Sylius\Component\Resource\Provider\LocaleProviderInterface;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * @author Anna Walasek <anna.walasek@lakion.com>
+ */
+class ResourceTranslationTypeSpec extends ObjectBehavior
+{
+    function let(LocaleProviderInterface $localeProvider, LocaleContextInterface $localeContext)
+    {
+        $this->beConstructedWith($localeProvider, $localeContext);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Bundle\ResourceBundle\Form\Type\ResourceTranslationsType');
+    }
+
+    function it_has_a_name()
+    {
+        $this->getName()->shouldReturn('sylius_translations');
+    }
+
+    function it_has_a_parent_type()
+    {
+        $this->getParent()->shouldReturn('collection');
+    }
+
+    function it_builds_form($localeProvider, FormBuilderInterface $builder)
+    {
+        $localeProvider->getAvailableLocales()->willReturn(['EN_en', 'PL_pl']);
```

Comment:
I know it's a placeholder, but `EN_en` looks weird, same for `PL_pl` -> `pl_PL` :)